server/boot/extend-models.js

```diff
@@ -18,4 +20,27 @@ module.exports = function(server) {
   serviceMetric(server.models.ServiceMetric);
   serviceProcess(server.models.ServiceProcess);
   gateway(server.models.Gateway);
+  var modelsToWatch = [
+    {name: 'agenttrace', instance: agentTrace},
+    {name: 'executor', instance: executor},
+    {name: 'expressusagerecord', instance: expressUsageRecord},
+    {name: 'profiledata', instance: profileData},
+    {name: 'serviceinstance', instance: serviceInstance},
+    {name: 'serverservice', instance: serverService},
+    {name: 'servicenetric', instance: serviceMetric},
+    {name: 'serviceprocess', instance: serviceProcess},
+    {name: 'gateway', instance: gateway},
+  ];
+  server.on('ready', function(dbWatcher) {
```

Comment:
Where does this event come from and at what point is it emitted? The watchers will need to be set up after DB, Driver etc are crated but before the driver and gateway re-connect calls. Perhaps this should be an method on the mesh server that can be called when we need it?